src/themes/base/colors.ts

```diff
@@ -0,0 +1,128 @@
+import { ColorPalette, ContextualColors, NaturalColors } from '../types'
+
```

Comment:
Not sure whether we want some base colors to be available at the level of base theme or not. Let's find the right solution.